.github/workflows/CI-BUILD.yml

```
@@ -0,0 +1,172 @@
+# Continuous Integration workflow for building, the project
+#
+# Jobs included:
+# - BUILD: Ensures the project compiles correctly
+# - BOOTSTRAP: Tests installation across Python versions and locales
+#
+# Required Secrets:
+# NONE
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: *
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 14: did not find expected alphabetic or numeric character

(syntax-check)